### ISSUE TITLE
Game Crash

### ISSUE THREAD

**Issue by Saxxonknight** [The-Balthazar/BrewLAN#20]
Similar to issues as before.  I have dmp files, they won't upload here (as I noted on Montsagott's thread the files are in code of some sort).

The last crash I noted happened right as I completed a nuke def.  Maybe coincidence, but it sure looked like the completion of that unit crashed the game.

**Comment by The-Balthazar** [comment]
The log doesn't show any errors or warnings, just that it couldn't find the image of sel0319, and that it had just loaded the scripts for uel0304, ueb2304, ssl0319, and aifmissileserpentine01; ssl0319 being my Seraphim T3 Field Engineer. That is unlikely to be the original cause of the crash; the script is basically nothing: [SSL0319_Script.lua](https://github.com/The-Balthazar/BrewLAN/blob/master/mods/BrewLAN/units/SSL0319/SSL0319_Script.lua)

**Comment by Saxxonknight** [comment]
Yeah I wish I knew how to get a more precise accounting of whats going wrong.  BrewLAN was only mod running, its fairly consistent that I get into the game around 30-60 minutes of game time and it crashes.  I could be something the AI is doing as well, generally I see them making gantrys and the like.

**Comment by The-Balthazar** [comment]
I am fairly sure it's an engine issue compounded by some of the things BrewLAN does. Like how Oblivion used to have an issue recycling data values, which was pushed into overdrive with the Shivering Isles. I did find some minor correlation between irrelevant log spam and the crashes, but you can only go so far with that.

**Comment by Saxxonknight** [comment]
I don't know enough about computers and coding to perhaps setup a goat computer that runs the game while you can have essentially a live error log running on another to forecast and catch the issue.

I do know that its generally after at least 30 minutes of game time meaning that certain tech levels and units are achieved or made, and likely a certain number of units in play.  All the games I have that last that long involve 6-7 AIs vs myself.

That irrelevant log spam may mean something that handled those items got bolloxed up, ie they are symptoms not the cause.  Say the code that loads those .lua's gets corrupted in memory by something that overwrites it, or changes it in some manner to enact some of your code in BrewLan.

I know that's shooting in the dark, but I don't know otherwise how to shine a light on the problem if the logs won't point to it.  Have to use them to point around the problem and find it in the middle.

**Comment by The-Balthazar** [comment]
I have separated out a bunch of the Ai control things to see if they are the cause.

**Comment by Saxxonknight** [comment]
Yeah I tried a couple games and still had same issue, game would crash after 30-60 minutes of play.  I had one other that I'd saved, but then the saved game would fail to load properly.  Unsure if that is related (ie a cumulative error that causes that crash later, that prevented the saved game from loading properly in latent form

**Comment by Montagsgott** [comment]
Hi Saxxonknight, hi Balthazar,

maybe it's related to FAF? As far as i know FAF changes some things in the engine, at least upgrades some effects.
My best guess would be to copy the settings of a player who can run the mod flawlessy?

**Comment by Montagsgott** [comment]
I've read about your leg, Balthazar. Get well soon!
